### PR TITLE
Initial list for CODEOWNERS

### PR DESCRIPTION
**Pull request by kalafut**
An initial cut at some CODEOWNERS rules.

### CODE REVIEW

**Review by calvn — APPROVED** [review]
Since we already have a `.github` directory, can we move this into there?

**Comment by kalafut** [comment]
I debated the location... Terraform has theirs at the root. If we want it to also serve as a more visible bit of documentation I think it's better there, but I'm fine with either location really.

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Each line is a file pattern followed by one or more owners.
+# More on CODEOWNERS files: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+
+/plugins/             @hashicorp/vault-ecosystem
```
Comment (pcman312) [inline CODEOWNERS:5]: nit: what do you think about alphabetizing the list? I'm thinking it will help with future additions to the list.

**Review by pcman312 — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (kalafut) [reply CODEOWNERS]: Done

**Review by pcman312 — APPROVED** [review]

**Comment by pcman312** [comment]
re: file location: I like it better at the root because it makes it more visible rather than putting it in a metaphorical corner.

**Pull request merged**